### PR TITLE
Changed the following:

### PR DESCRIPTION
**Pull request by slavabez**
1. Added basic ReadMe with some instructions on how to run.
2. Added a separate config (sample-config) to prevent committing sensitive data (file structure)
3. Changed the URL pulled to the url of the source image via data.preview.images object instead of the URL from data.url, which sometimes links to a page instead of an image. Seems to always work, but needs some more testing
4. Added minimum file size to the config with width/height with checks in the script
5. Added package-lock.json for NPM5 users

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# Reddit Wallpaper Finder script in NodeJS
```
Comment (the-pat) [inline README.md]: Could you change this header to the repo name (`reddit-node-wallpaper`)?

File: index.js [hunk 2]
```diff
@@ -41,13 +41,15 @@ const change = () => {
...
-                                                   !post.data.over_18)
-                                   .map(post => ({
-                                       url: post.data.url,
-                                       domain: post.data.domain.toLowerCase(),
-                                   }));
+        post.data &&
+        config.domains.includes(post.data.domain.toLowerCase()) &&
+        !post.data.over_18 &&
+        post.data.preview.images[0].source.width >= config.minimum_image_size.width &&
+        post.data.preview.images[0].source.height >= config.minimum_image_size.height)
```
Comment (the-pat) [inline index.js:57]: This is pretty slick!

File: sample_config.json [hunk 3]
```diff
@@ -0,0 +1,14 @@
+{
```
Comment (the-pat) [inline sample_config.json]: Can we rename the file to `config.example.json`?

**Review by the-pat — CHANGES_REQUESTED** [review]
Awesome! Thanks for working on this.

I gave minor feedback (I am willing to merge it without the those changes), but I would like the merge conflicts to get resolved before approving and merging.

**Review by the-pat — APPROVED** [review]

**Comment by slavabez** [comment]
That was quick, thanks! Nice app by the way, found a few pretty sick wallpapers

**Pull request merged**